test/box/gh-5928-several-listen-sockets.result

```diff
@@ -0,0 +1,429 @@
+    for i = 1, #server_addresses_table do
+        local t = str_split(server_addresses_table[i], "?")
+        if check_connection(t[1]) ~= true then
+            connection_result = false
+        end
+    end
+    return connection_result
+end;
+ | ---
+ | ...
+function check_connection_for_table_uri(server_addresses_table)
+    local connection_result = true
+    for _, server_addresses_str in pairs(server_addresses_table) do
+        connection_result = check_connection_for_string_uri(server_addresses_str)
+    end
+    return connection_result
+end;
+ | ---
+ | ...
+function check_connection_ex()
+    local server_addresses = test_run:eval('test', 'return box.cfg.listen')[1]
+    if type(server_addresses) == 'table' then
+        return check_connection_for_table_uri(server_addresses)
+    else
+        return check_connection_for_string_uri(server_addresses)
+    end
+end;
+ | ---
+ | ...
+test_run:cmd("setopt delimiter ''");
+ | ---
+ | - true
+ | ...
+
+test_run:cmd(string.format("start server test with args=\"%s\"", 1))
+ | ---
+ | - true
+ | ...
+default_server_addr = test_run:eval('test', 'return box.cfg.listen')[1]
+ | ---
+ | ...
+test_run:cmd("stop server test")
+ | ---
+ | - true
+ | ...
+
+-- Checks that we able to open several listening sockets in several threads
+-- Checks that all unix socket path successfully deleted after after listening is stop
+connection_result = true
+ | ---
+ | ...
+graceful_release = true
+ | ---
+ | ...
+test_run:cmd("setopt delimiter ';'")
+ | ---
+ | - true
+ | ...
+for thread_count = 1, 3 do
+    for listen_count = 1, 2 do
+    	test_run:cmd(string.format("start server test with args=\"%s\"", thread_count))
+        local addr = prepare_several_listen_uri(default_server_addr, listen_count)
+        test_run:eval('test', string.format("box.cfg{ listen = \'%s\' }", addr))
+        local server_addresses_str = test_run:eval('test', 'return box.cfg.listen')[1]
+        local server_addresses_table = str_split(server_addresses_str, ", ")
+        for i = 1, #server_addresses_table do
+            if check_connection(server_addresses_table[i]) ~= true then
+                connection_result = false
+            end
+        end
+        test_run:eval('test', string.format("box.cfg{ listen = \'%s\' }", default_server_addr))
+        server_addresses_str = test_run:eval('test', "return box.cfg.listen")[1]
+        assert(default_server_addr == server_addresses_str)
+        test_run:cmd("stop server test")
+        for i = 1, #server_addresses_table do
+            if fio.path.exists(server_addresses_table[i]) then
+                graceful_release = false
+            end
+        end
+        if fio.path.exists(server_addresses_str) then
+                graceful_release = false
+        end
+    end
+end
+test_run:cmd("setopt delimiter ''");
+ | ---
+ | ...
+assert(connection_result == true)
+ | ---
+ | - true
+ | ...
+assert(graceful_release == true)
```

Comment:
You don't need `== true` for booleans. They are already booleans. You can use `assert(connection_result)`. Although the names clearly do not look like flags. For flags you need to use prefixes such as `is`, `has`, `does`, etc.